golang/testsuite/networks_impl/test_network.go

```diff
@@ -61,13 +64,22 @@ func (network *TestNetwork) SetupDatastoreAndTwoApis() error {
 		return stacktrace.NewError("Cannot add API services to network; one or more API services already exists")
 	}
 
-	configFactory := datastore.NewDatastoreContainerConfigFactory(network.datastoreServiceImage)
-	datastoreServiceContext, hostPortBindings, err := network.networkCtx.AddService(datastoreServiceId, configFactory)
+	containerCreationConfig := services.NewContainerCreationConfigBuilder(
```

Comment:
@leoporoli worth extracting this into a helper function, so this function isn't so enormous?